packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -126,9 +139,11 @@ export class Flow implements LoadableComponent {
   updateFlowProps = (): void => {
     const { el, items } = this;
 
-    const newItems: HTMLCalciteFlowItemElement[] = Array.from(
-      el.querySelectorAll("calcite-flow-item")
-    ).filter((flowItem) => flowItem.closest("calcite-flow") === el) as HTMLCalciteFlowItemElement[];
+    const newItems = Array.from<FlowItemLikeElement>(
+      el.querySelectorAll(
+        `calcite-flow-item${this.customItemSelectors ? `,${this.customItemSelectors}` : ""}`
```

Comment:
nitpick: since `this.customItemSelectors` is used twice, might be nice to include it here:

```
const { el, items, customItemSelectors } = this;
```